index.bs

```diff
@@ -1074,6 +1085,221 @@ spec: webidl
     </p>
   </section>
 </section>
+<section>
+  <h2 id="automation">
+    Automation
+  </h2>
+  <p>
+    For the purposes of user-agent automation and application testing, this
+    document defines the following <a>errors</a>, algorithms, and <a>extension
+    commands</a> for the [[WebDriver]] specification.
```

Comment:
I believe this is expected. See, for example, [the current rendering for the text `[[clipboard-apis]]`](https://w3c.github.io/permissions/#clipboard).